software/controller/lib/core/sensors.h

```diff
@@ -56,50 +45,35 @@ class Sensors {
  public:
   Sensors();
 
-  // Perform some initial sensor calibration.  This function should
+  // Performs general init.  This function should
   // be called on system startup before any other sensor functions
   // are called.
+  void init(Frequency cpu_frequency);
+
+  // Performs calibration of the sensors. For now, since sensor calibration are not
+  // stored in EEPROM, this function should be called immediately after init, but
+  // it shouldn't stay that way (to accomodate for ventilator restart while linked
+  // to a patient)
```

Comment:
Maybe make this an explicit `TODO` so it is easily findable as a known "bug"

Reply:
yes